forge-gui/res/adventure/Shandalar/world/enemies.json

```diff
@@ -3366,7 +3366,7 @@
 	"name": "Copper Host Brutalizer",
 	"sprite": "sprites/copperhostbrutalizer.atlas",
 	"deck": [
-		"deckscopperhostbrutalizer.json"
```

Comment:
The .json decks are still in the /decks folder, are these intended to be reused later or can these be removed as well ?